Tests on aarch64 + cpython3.10-dev have started failing

Is this due to a stale cache? For instance, [here](https://app.travis-ci.com/github/numpy/numpy/jobs/559481128#L3307)
```
./../builds/venv/lib/python3.10/site-packages/mypy/types.py:14: in <module>

    import mypy.nodes

../../builds/venv/lib/python3.10/site-packages/mypy/nodes.py:14: in <module>

    import mypy.strconv


../../builds/venv/lib/python3.10/site-packages/mypy/strconv.py:10: in <module>

    from mypy.visitor import NodeVisitor
../../builds/venv/lib/python3.10/site-packages/mypy/visitor.py:18: in <module>

    class ExpressionVisitor(Generic[T]):

/opt/python/3.10-dev/lib/python3.10/typing.py:287: in inner

    return cached(*args, **kwds)


/opt/python/3.10-dev/lib/python3.10/typing.py:1166: in __class_getitem__

    return _GenericAlias(cls, params)


/opt/python/3.10-dev/lib/python3.10/typing.py:874: in __init__

    self.__parameters__ = _collect_type_vars(params)

../../builds/venv/lib/python3.10/site-packages/typing_extensions.py:146: in _collect_type_vars

    if _should_collect_from_parameters(t):

../../builds/venv/lib/python3.10/site-packages/typing_extensions.py:120: in _should_collect_from_parameters

    t, (typing._GenericAlias, _types.GenericAlias, _types.UnionType)

E   AttributeError: module 'types' has no attribute 'UnionType'
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -26,7 +26,7 @@ cache:
 
 jobs:
   include:
-    - python: 3.8
+    - python: "3.8"
       os: linux
       arch: ppc64le
       env:
@@ -35,7 +35,7 @@ jobs:
        - NPY_USE_BLAS_ILP64=1
        - ATLAS=None
 
-    - python: 3.8
+    - python: "3.8"
       os: linux
       arch: s390x
       env:
@@ -44,7 +44,7 @@ jobs:
        - NPY_USE_BLAS_ILP64=1
        - ATLAS=None
 
-    - python: 3.8
+    - python: "3.8"
       os: linux
       arch: arm64
       virt: vm
@@ -53,7 +53,7 @@ jobs:
        - DOWNLOAD_OPENBLAS=1
        - ATLAS=None
 
-    - python: 3.10-dev
+    - python: "3.10"
       os: linux
       arch: arm64
       virt: vm

